### ISSUE TITLE
host on https://discovery.rust-embedded.org

### ISSUE THREAD

**Issue by japaric** [rust-embedded/discovery#101]
After moving this repo into the rust-embedded org we should host it under the rust-embedded.org domain as well.

cc @jamesmunns

**Comment by jamesmunns** [comment]
Hey @japaric, I'd suggest we get all the books (and anything else) moved over to the rust-embedded space, and once there is Github pages/Travis CI set up, we can talk to Nick Stevens about updating the DNS/CNAME records.

**Comment by jamesmunns** [comment]
Linking to https://github.com/rust-embedded/wg/issues/156 to keep track of things

**Comment by japaric** [comment]
Triage: blocked on rust-embedded/wg#156

**Comment by japaric** [comment]
Closing in favor of rust-embedded/wg#208
